src/client/Table.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import data from './data.json';
+
+function TableItem(props){
```

Comment:
Generally, files containing multiple `classes` or `components` can often result in a less navigable and poorly structured codebase. Best practice is to place your component in one file and keep each file limited to a single responsibility.